### DISCUSSION TITLE
"Error code 2147024888"?

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [Klocman/Bulk-Crap-Uninstaller#553] [General]
BCU v5. 7 

Hi there,

After attempting to uninstall multiple previous-version "Intel Graphics Driver Software" packages, they returned the error message, "Failed - Uninstaller returned error code: -2147024888"... 

![Screenshot 2023-11-07 165826](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/59787873/2a08b117-d34f-44c5-9937-c8f244c58cc5)

...Not much turned up in a search; wondering if anyone is familiar with the error or maybe has other insights as to why it didn't work? 

Prior to attempting, Intel Support informed me that these packages are what remain on the system when the end user doesn't select "clean install" during manual graphic software updates, so it seemed as though using BCU to remove them would be a straightforward scenario. 

Thanks to the developer(s) for your efforts on one of my go-to apps. 

Alex

**Comment by Klocman** [comment]
Do these packages still actually exist after a list refresh? They might have been removed by the uninstallations that did succeed.

The error code is probably meant to be a hex code, after converting int -> uint -> 0x80070008 which seems to be a fairly generic failure, possibly out of memory.

**Reply by GitAlexToTheGreek** [reply]
Hi Klocman,

Thanks for the follow up. 

Yes, they're still listed...

![1-1](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/59787873/eb4870f0-d778-4ef5-8a25-88afd37c5f38)

Thx for the error code explanation.

**Reply by Klocman** [reply]
You can try removing them again, it's not really a BCU problem, the uninstallers themselves are failing. You might need to use DDU.

**Reply by GitAlexToTheGreek** [reply]
Worked! 😀

<img width="963" alt="2" src="https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/59787873/b84745a3-69e9-486e-aa12-8097877aa80a">

Much obliged!

**Reply by GitAlexToTheGreek** [reply]
😬 Back for more 😬

<img width="905" alt="3" src="https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/59787873/7feccedc-b1a8-493e-a02b-d6bfc41e93f0">

So, after DDU, and screenshot'ing BCU (in the previous post) I installed the latest Intel driver without issue.

Checked BCU again, and noticed the 'command center' (which was not present in the installer checklist) which I don't need and runs in the background, so definitely want to uninstall it...

Despite selecting the checkbox in BCU, an uninstaller is not recognized. 🤷

Did DDU remove the ability to uninstall Intel stuff across the board going forward?

If you can recommend what to do next in order have BCU normally uninstall Intel things again, I'd greatly appreciate it (thx for your patience).

**Reply by Klocman** [reply]
DDU only removes drivers, it doesn't affect anything that happens afterwards.

**Comment by GitAlexToTheGreek** [comment]
Ok, wrapping up...

Windows Update installed a ton of Intel drivers, tried the 'command center' uninstallation in BCU again, and it worked normally.

✌️

**Reply by Klocman** [reply]
Good to hear you got it resolved.
